### PR TITLE
ForemanGoogle as a main source

### PR DESCRIPTION
**Pull request by stejskalleos**
Follow up PR for the removal of GCE
code from the Foreman core [0]

- Remove TemporaryPrependPath
- Extend API documentation

[0] https://github.com/theforeman/foreman/pull/9272

### CODE REVIEW

**Comment by stejskalleos** [comment]
Tests are failing, https://github.com/theforeman/foreman/pull/9272 must be merged before

File: app/models/foreman_google/gce.rb [hunk 1]
```diff
@@ -85,10 +85,14 @@ def destroy_vm(uuid)
       true
     end
 
-    def available_images(filter: nil)
+    def available_images(filter: filter_for_images)
       client.images(filter: filter)
     end
 
+    def filter_for_images
```
Comment (adamruzicka) [inline app/models/foreman_google/gce.rb:92]: Is this expected to be overridden from somewhere?

Comment (stejskalleos) [reply app/models/foreman_google/gce.rb:92]: Yop, it's overridden from the `foreman_theme_satellite` plugin

**Comment by stejskalleos** [comment]
@adamruzicka https://github.com/stejskalleos/hammer-cli-foreman-google/pull/1 & https://github.com/theforeman/foreman/pull/9272 have been merged & tests are green, can we merge it?

**Comment by adamruzicka** [comment]
Thank you @stejskalleos !

**Pull request merged**